shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/process/spi/ExecuteProcessReporter.java

```diff
@@ -31,9 +31,8 @@
      * Report the summary of this task.
      * @param context context
      * @param executionGroupContext execution group context
-     * @param constants constants
      */
-    void report(SQLStatementContext<?> context, ExecutionGroupContext<? extends SQLExecutionUnit> executionGroupContext, ExecuteProcessConstants constants);
+    void report(SQLStatementContext<?> context, ExecutionGroupContext<? extends SQLExecutionUnit> executionGroupContext);
```

Comment:
Is there no possibility that `ExecuteProcessConstants` is another value? How about report stoping, resuming status?

Reply:
Currently, `ExecutionGroupContext` constructor just use `ExecuteProcessConstants.EXECUTE_STATUS_START` to construct `ExecuteProcessUnit`.
I'll update it.